style.css

```diff
@@ -16,10 +16,12 @@ h1 {
     display: flex;
     justify-content: center;
     align-items: center;
+    margin: 10px;
 }
 
-.landingLeft {
+.stopwatchContainer {
     width: 25%;
```

Comment:
width 25% is quite a random width, it should probably be something else (or no set width)